src/handler/unarchive.ts

```diff
@@ -42,7 +42,7 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     }
 
     const { primaryVrm } = (record as { primaryVrm?: string });
-    const hasUnarchivedRecords = (await searchByCriteria(SearchCriteria.SYSTEM_NUMBER, record.systemNumber))
+    const hasUnarchivedRecords = (await searchByCriteria(SearchCriteria.PRIMARYVRM, primaryVrm as string))
```

Comment:
surely this shifts the problem onto VRM where if it has changed you end up with the same issue? Or am I misunderstanding

Reply:
Can't you only have one current / provisional record per vehicle tho?